scripts/HGDP_SampleSelection.ipynb

```diff
@@ -0,0 +1,1716 @@
+{
```

Comment:
<p>Another way to do this is with f-strings (https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python):</p><p> </p><pre class="ql-syntax" spellcheck="false">&gt;&gt;&gt; x = 123
&gt;&gt;&gt; print(f'After filtering, {x} samples remain')
After filtering, 123 samples remain
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/populationgenomics/ancestry/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>